From 4e84b336e5a53030b6a80702dc296c082a14d7b0 Mon Sep 17 00:00:00 2001
From: Daniel Nachun <daniel.nachun@gmail.com>
Date: Fri, 3 Jan 2025 13:30:18 -0600
Subject: [PATCH] minimal ucrt libs

---
 src/Makevars.ucrt | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/src/Makevars.ucrt b/src/Makevars.ucrt
index be127cd..e69de29 100644
--- a/src/Makevars.ucrt
+++ b/src/Makevars.ucrt
@@ -1,14 +0,0 @@
-PKG_LIBS = -ludunits2 -lexpat
-
-all: clean winlibs
-
-winlibs:
-	mkdir -p ../inst
-	rm -Rf ../inst/share
-	mkdir -p ../inst/share/udunits
-	cp -r $(R_TOOLS_SOFT)/share/udunits/* ../inst/share/udunits
-
-clean:
-	rm -Rf $(SHLIB) $(OBJECTS)
-
-.PHONY: all winlibs clean
-- 
2.47.1

